libs/utils/utils.js

```diff
@@ -330,3 +344,42 @@ export function updateObj(obj, defaultObj) {
   return obj;
 }
 
+export function isNullish(thing) {
+  const type = typeof thing;
+  return (
+    type === 'undefined'
+    || (type === 'object' && thing === null)
+    || (Number.isNaN(thing))
+    || (type === 'string' && thing === '')
+  );
+}
+
+export function isEmptyObject(obj) {
+  let isEmpty = false;
+
+  if (typeof obj !== 'object' || isNullish(obj)) {
```

Comment:
```suggestion
  if (typeof obj !== 'object' || !obj) {
```

Reply:
I removed `isEmptyObject` altogether because it we're no longer using it